mono/metadata/object.c

```diff
@@ -7190,26 +7190,20 @@ mono_string_to_utf8_mp_ignore (MonoMemPool *mp, MonoString *s)
  * This is a temporary helper until our string implementation
  * is reworked to always include the null-terminating char.
  */
-mono_unichar2*
+gunichar2*
```

Comment:
Even if the end type is the same (uint16_t), it looks weird that the signature is `mono_unichar2* mono_string_to_utf16 (MonoString *s)` in `object.h` (which is a public header and thus cannot be modified) , and `gunichar2* mono_string_to_utf16 (MonoString *s)` here. So please keep `mono_unichar2` here.

Reply:
I was preferring the shorter name, before learning that glib might not be ok. Ok.